citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -5321,10 +5321,14 @@ protected ContentResource createTemporaryResource(ResourceToolActionPipe pipe)
 
 	protected String validateURL(String url) throws MalformedURLException
 	{
-		if (url == null || url.trim().equals (""))
+		if (url == null)
 		{
 			throw new MalformedURLException();
 		}
+		if(url.equals(""))
```

Comment:
What's the difference between a null and empty string for a url here?  Does null mean something has gone wrong and empty string that they've deleted a url?